### PR TITLE
Added customized message on home page using .md file.

### PR DESCRIPTION
**Pull request by NishaSharma14**

### CODE REVIEW

File: app/packs/src/components/WelcomeMessage.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    return null;
+  }
+}
+
+class WelcomeMessage extends Component {
+  state = {
+    post: null
+  }
+  componentDidMount () {
+    fetch('welcome-message.md')
```
Comment (Allenskywalker92) [inline app/packs/src/components/WelcomeMessage.js:19]: @NishaSharma14  you fetch the wrong file name.

File: app/packs/src/components/WelcomeMessage.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Jumbotron } from 'react-bootstrap';
+import ReactMarkdown from 'react-markdown';
+
+function Message(props) {
+  const input = props.post;
+  if (input != null && input !== undefined)  {
+    return <Jumbotron> <ReactMarkdown children={input} /> </Jumbotron>;
```
Comment (Allenskywalker92) [inline app/packs/src/components/WelcomeMessage.js:8]: Please format code this file.

**Review by Allenskywalker92 — CHANGES_REQUESTED** [review]

File: app/packs/src/components/WelcomeMessage.js (hunk 1 above)
Comment (NishaSharma14) [reply app/packs/src/components/WelcomeMessage.js:19]: filename is correct. But this file should be optional and its in gitignore.

File: app/packs/src/components/WelcomeMessage.js (hunk 2 above)
Comment (NishaSharma14) [reply app/packs/src/components/WelcomeMessage.js:8]: ok..I will.

**Review by Allenskywalker92 — APPROVED** [review]

**Pull request merged**